src/plugins/management/public/components/landing/solution_empty_prompt.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+import React, { type FC } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { EuiButton, EuiLink } from '@elastic/eui';
+import { type CoreStart } from '@kbn/core/public';
+import { KibanaPageTemplate } from '@kbn/shared-ux-page-kibana-template';
+
+interface Props {
+  kibanaVersion: string;
+  coreStart: CoreStart;
+}
+
+const IndicesLink: FC<{ coreStart: CoreStart }> = ({ coreStart }) => (
+  <EuiLink
+    href={coreStart.application.getUrlForApp('management', { path: 'data/index_management' })}
+    data-test-subj="managementLinkToIndices"
+  >
+    {i18n.translate('management.landing.subhead.indicesLink', {
+      defaultMessage: 'indices',
+    })}
+  </EuiLink>
+);
+
+const DataViewsLink: FC<{ coreStart: CoreStart }> = ({ coreStart }) => (
+  <EuiLink
+    href={coreStart.application.getUrlForApp('management', { path: 'kibana/dataViews' })}
+    data-test-subj="managementLinkToDataViews"
+  >
+    {i18n.translate('management.landing.subhead.dataViewsLink', {
+      defaultMessage: 'data views',
+    })}
+  </EuiLink>
+);
+
+const IngestPipelinesLink: FC<{ coreStart: CoreStart }> = ({ coreStart }) => (
+  <EuiLink
+    href={coreStart.application.getUrlForApp('management', { path: 'ingest/ingest_pipelines' })}
+    data-test-subj="managementLinkToIngestPipelines"
+  >
+    {i18n.translate('management.landing.subhead.ingestPipelinesLink', {
+      defaultMessage: 'ingest pipelines',
+    })}
+  </EuiLink>
+);
+
+const UsersLink: FC<{ coreStart: CoreStart }> = ({ coreStart }) => (
+  <EuiLink
+    href={coreStart.application.getUrlForApp('management', { path: 'security/users' })}
+    data-test-subj="managementLinkToUsers"
+  >
+    {i18n.translate('management.landing.subhead.usersLink', {
+      defaultMessage: 'users',
+    })}
+  </EuiLink>
+);
+
+export const SolutionEmptyPrompt: FC<Props> = ({ kibanaVersion, coreStart }) => {
+  return (
+    <KibanaPageTemplate.EmptyPrompt
+      data-test-subj="managementHomeSolution"
+      iconType="managementApp"
+      title={
+        <h1>
+          <FormattedMessage
+            id="management.landing.solution.header"
+            defaultMessage="Stack Management {version}"
+            values={{ version: kibanaVersion }}
+          />
```

Comment:
Should we exclude kibana version particularly for serverless projects, to be in line with the direction we've taken for other instances where kibana version previously would get displayed?

Reply:
The empty prompt is not shown in serverless. In serverless we display the management cards.